src/backend/joanie/core/serializers/admin.py

```diff
@@ -350,9 +369,26 @@ class Meta:
             "certificate_definition",
             "contract_definition",
             "target_courses",
+            "certification_level",
+            "teachers",
+            "skills",
         ]
         read_only_fields = ["id"]
 
+    def validate_certification_level(self, value):
```

Comment:
I might be wrong but I don't see the test of this validation ?

Reply:
Oh right. I'm pretty sure to wrote 3 tests about that... 🤔